python/cuda_parallel/pyproject.toml

```diff
@@ -17,17 +17,41 @@ classifiers = [
 ]
 requires-python = ">=3.9"
 dependencies = ["cuda-cccl", "numba>=0.60.0", "cuda-python==12.*"]
-dynamic = ["version", "readme"]
+dynamic = ["version"]
+readme = { file = "README.md", content-type = "text/markdown" }
 
 [project.optional-dependencies]
 test = ["pytest", "pytest-xdist", "cupy-cuda12x", "typing_extensions"]
 
 [project.urls]
 Homepage = "https://developer.nvidia.com/"
 
-[tool.setuptools.dynamic]
-version = { attr = "cuda.parallel._version.__version__" }
-readme = { file = ["README.md"], content-type = "text/markdown" }
+[tool.scikit-build]
+minimum-version = "build-system.requires"
+build-dir = "build/{wheel_tag}"
+
+[tool.scikit-build.cmake]
+version = ">=3.21"
+args = []
+build-type = "Release"
+source-dir = "."
+
+[tool.scikit-build.ninja]
+version = ">=1.11"
+make-fallback = true
+
+[tool.scikit-build.wheel]
+py-api = "py3"
+
+[tool.scikit-build.wheel.packages]
```

Comment:
Question: what is this configuration option?

Reply:
See https://scikit-build-core.readthedocs.io/en/latest/configuration.html#customizing-the-built-wheel 

By default scikit-build-core expects sources to be located in `src/<pkg_name>`, `python/<pkg_name>`, or in `<pkg_name>` folders. In both packages it is not located there, and content of packages is placed into `cuda/` placeholder package in the prefix layout. This configuration option allows for that to happen.